pallets/gear/src/manager.rs

```diff
@@ -183,7 +186,10 @@ where
                     })
                     .unwrap_or_default();
 
-                log::trace!("Dispatch outcome trap: {:?}", message_id);
+                log::debug!(
+                    "Dispatch outcome trap: 0x{:?}",
+                    HexDisplay::from(&message_id.as_ref().to_vec())
```

Comment:
Remove it from everywhere, please. Our ids are always displayed and debugged as hexes

Reply:
Sure. This is a legacy from more than 2 months back - ids were not displayed as hex back then. Will clean it up anyway